shared-bindings/sdcardio/SDCard.c

```diff
@@ -69,10 +69,11 @@
 //|             import os
 //|
 //|             import board
+//|             import busio
 //|             import sdcardio
 //|             import storage
 //|
-//|             sd = sdcardio.SDCard(board.SPI(), board.SD_CS)
+//|             sd = sdcardio.SDCard(busio.SPI(), board.SD_CS)
```

Comment:
Doesn't busio.SPI() require the clock, mosi and miso arguments? So this should be something like:  
`sd = sdcardio.SDCARD(busio.SPI(board.SCK,board.MOSI,board.MISO), board.SD_CS)`   

I tested this version of the code using a board that had board.SPI() defined and it gave a "ValueError: IO40 in use" (IO40 is the clock pin).

Perhaps both the busio and board lines should included in the example code with one of them commented out:
```py
sd = sdcardio.SDCard(board.SPI(), board.SD_CS)
# If board.SPI is not defined on the board being used
# sd = sdcardio.SDCard(busio.SPI(board.SCK,board.MOSI,board.MISO), board.SD_CS)
```

Reply:
Good point, hadn't thought of that!